### PR TITLE
Initializes tmpfs_alloc_node() implementation

### PR DESCRIPTION
**Pull request by ultimaweapon**

### CODE REVIEW

File: src/kernel/src/fs/host/mod.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ fn mount(mount: &mut Mount, mut opts: HashMap<String, Box<dyn Any>>) -> Result<(
         todo!("update root FS mounting");
     }
 
-    flags.set(MountFlags::MNT_LOCAL, true); // TODO: Check if this flag has been set for exfatfs.
+    flags.set(MountFlags::MNT_LOCAL, true);
```
Comment (SuchAFuriousDeath) [inline src/kernel/src/fs/host/mod.rs:45]: This could be flags.insert(...)

File: src/kernel/src/fs/tmp/mod.rs [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 fn mount(mnt: &mut Mount, mut opts: HashMap<String, Box<dyn Any>>) -> Result<(), Box<dyn Errno>> {
-    if mnt.flags().intersects(MountFlags::MNT_UPDATE) {
+    // Check flags.
+    let mut flags = mnt.flags_mut();
+
+    if flags.intersects(MountFlags::MNT_UPDATE) {
         return Err(Box::new(MountError::UpdateNotSupported));
     }
 
+    flags.set(MountFlags::MNT_LOCAL, true);
```
Comment (SuchAFuriousDeath) [inline src/kernel/src/fs/tmp/mod.rs:23]: Same thing

**Review by SuchAFuriousDeath — CHANGES_REQUESTED** [review]

**Comment by SuchAFuriousDeath** [comment]
Nice :D 
image

**Comment by ultimaweapon** [comment]
The good news is those message will be squashed when merged.

**Review by SuchAFuriousDeath — APPROVED** [review]

**Pull request merged**